pom.xml

```diff
@@ -17,6 +17,21 @@
   </scm>
 
     <repositories>
+            <repository>
+            <id>central</id>
+            <name>IAD Artifacts-releases</name>
+            <url>https://artifacts.rackspace.net:443/artifactory/cloudfeeds-maven-local</url>
+        </repository>
+        <repository>
+            <id>central2</id>
+            <name>jcenter-remote-cache</name>
+            <url>https://artifacts.rackspace.net:443/artifactory/jcenter-remote-cache</url>
+        </repository>
```

Comment:
JFrog is being deprecated so these artifactory endpoints will become defunct remove them and if necessary replace them.